nixos/modules/services/networking/tailscale.nix

```
@@ -49,6 +49,15 @@ in {
         When set to `server` or `both`, IP forwarding will be enabled.
       '';
     };
+
+    authKeyFile = mkOption {
+      type = with types; nullOr path;
```

Let's stay consistent with other existing options:

```suggestion
      type = types.nullOr types.path;
```